### PR TITLE
Add in-play golden frames

### PR DESCRIPTION
**Pull request by andburn**
- Adds images for golden in-play minions
- Adds class colours to golden cards (*uses css filters so the colours aren't 100% correct, but better than 30 extra images*).

### CODE REVIEW

**Comment by beheh** [comment]
Thanks, looking very good. Will fully review and land later this week.

File: less/entities.less [hunk 1]
```diff
@@ -360,3 +360,46 @@
...
+.priest-color {
+	visibility: hidden;
+}
+
+.rogue-color {
+	filter: hue-rotate(104deg) saturate(0%) brightness(80%);
+}
+
+.shaman-color {
+	filter: hue-rotate(250deg) saturate(75%) brightness(80%);
```
Comment (beheh) [inline less/entities.less]: Color looks wrong, looks right for me with about `filter: hue-rotate(220deg) saturate(50%) brightness(100%);`

File: less/entities.less [hunk 2]
```diff
@@ -360,3 +360,46 @@
...
+.druid-color {
+	filter: hue-rotate(135deg) saturate(40%) brightness(95%);
+}
+
+.hunter-color {
+	filter: hue-rotate(150deg) saturate(80%) brightness(100%);
+}
+
+.mage-color {
+	filter: hue-rotate(303deg) saturate(88%) brightness(94%);
```
Comment (beheh) [inline less/entities.less]: Looks really wrong:

Wrong mage color

**Comment by andburn** [comment]
**Updated**. Did my best with improving the colours, paladin is completely wrong though - I just can't seem to get anything close to the colour. They look like this now:

display

**Comment by beheh** [comment]
:+1: Nice work. Landing this and revisiting the paladin color in #139.

**Comment by beheh** [comment]
Mage is also really broken.

**Pull request merged**